prowler/providers/aws/lib/service/service.py

```diff
@@ -113,4 +113,7 @@ def get_unknown_arn(self, region) -> str:
             >>> service.get_unknown_arn("us-east-1")
             "arn:aws:service:us-east-1:0123456789012:unknown"
         """
-        return f"arn:{self.audited_partition}:{self.service}:{region}:{self.audited_account}:unknown"
+        if region:
+            return f"arn:{self.audited_partition}:{self.service}:{region}:{self.audited_account}:unknown"
```

Comment:
You can do the conditional within the f-string, but not mandatory.

Reply:
Done